asset/css/search-editor.less

```diff
@@ -96,7 +116,7 @@
 
     &:hover i.icon:before {
       .rounded-corners(1em);
```

Comment:
I'd prefer
```
.rounded-corners(.25em)
```
because we don't have circular elements of this kind anywhere else in our UI.

Reply:
Except the close button in the modal right above it :laughing:

Reply:
I'll use .4em. That's the same everywhere else.